crates/stackable-certs/src/ca/mod.rs

```diff
@@ -68,6 +68,48 @@ pub enum Error {
     ParseAuthorityKeyIdentifier { source: x509_cert::der::Error },
 }
 
+/// Custom implementation of [`std::cmp::PartialEq`] because some inner types
+/// don't implement it.
+///
+/// Note that this implementation is restritced to testing because there is a
+/// variant that is impossible to compare, and will cause a panic if it
```

Comment:
nitpick: Seems like `is` is missing here:

```suggestion
/// variant that is impossible to compare, and will cause a panic if it is
```